compose.js

```diff
@@ -88,12 +88,12 @@ function mergeComposable(dstDescriptor, srcComposable) {
   combineProperty('configuration', assign);
   combineProperty('deepConfiguration', merge);
   if (Array.isArray(srcDescriptor.initializers)) {
-    if (!Array.isArray(dstDescriptor.initializers)) dstDescriptor.initializers = [];
-    srcDescriptor.initializers.forEach(i => {
-      if (isFunction(i) && dstDescriptor.initializers.indexOf(i) < 0) {
-        dstDescriptor.initializers.push(i);
+    dstDescriptor.initializers = srcDescriptor.initializers.reduce((result, init) => {
```

Comment:
I did test it. We have the special package `check-compose` which runs unit tests agains a "compose()" implementation. I have just updated the `check-compose` up to version two, - added two more tests to check that initializers do not duplicate.